src/openrct2-ui/windows/About.cpp

```diff
@@ -28,21 +28,16 @@ enum
     WINDOW_ABOUT_PAGE_RCT2,
 };
 
-enum WINDOW_ABOUT_WIDGET_IDX {
-    WIDX_BACKGROUND,
-    WIDX_TITLE,
-    WIDX_CLOSE,
-    WIDX_PAGE_BACKGROUND,
-    WIDX_TAB_ABOUT_OPENRCT2,
-    WIDX_TAB_ABOUT_RCT2,
-
-    WIDX_PAGE_START,
-
-    // About OpenRCT2
-    WIDX_CHANGELOG = WIDX_PAGE_START,
-
-    // About RCT2
-    WIDX_MUSIC_CREDITS = WIDX_PAGE_START,
+enum class WindowAboutWidgetIndex {
+    Background,
+    Title,
+    Close,
+    PageBackground,
+    TabAboutOpenRCT2,
+    TabAboutRCT2,
+    PageStart,
+    ChangeLog,
+    MusicCredits
```

Comment:
You changed the values of Changelog and MusicCredits. These were set to PageStart for a good reason.

Also, it's Changelog and not ChangeLog.